cmake/common.cmake

```diff
@@ -348,12 +348,80 @@ set(CCF_NETWORK_TEST_ARGS -l ${TEST_HOST_LOGGING_LEVEL} --worker-threads
                           ${WORKER_THREADS}
 )
 
+if("sgx" IN_LIST COMPILE_TARGETS)
```

Comment:
Is the plan to ultimately move this to new cmake file located near the plugin code so that the plugin is self contained?

Reply:
Yes.